bindings/csharp/Keystone.Net/Keystone.Net.csproj

```diff
@@ -1,28 +1,36 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Since we're not using features only available in .NET Standard 2.0 (AFAIK), I think we should try targeting multiple frameworks by replacing

```xaml
<TargetFramework>netstandard2.0</TargetFramework>
```

by

```xaml
<TargetFrameworks>netstandard1.1;netstandard2.0</TargetFrameworks>
```